develop/styleguide/python.rst

```diff
@@ -66,11 +65,16 @@ Based on code we love to look at (Pyramid, Requests, etc.), we allow the followi
            "an_exceptionally_long_string_of_characters",
        ]
 
- * Arguments on first line, directly after the opening parenthesis are forbidden when breaking lines.
- * The last argument line needs to have a trailing comma (to be nice to the next developer coming in to add something as an argument and minimize VCS diffs in these cases).
- * The closing parenthesis or bracket needs to have the same indentation level as the first line.
- * Each line can only contain a single argument.
- * The same style applies to dicts, lists, return calls, etc.
+* Arguments on first line, directly after the opening parenthesis are forbidden when breaking lines.
+
+* The last argument line needs to have a trailing comma (to be nice to the next developer coming in to add something as an argument and minimize VCS diffs in these cases).
+
+* The closing parenthesis or bracket needs to have the same indentation level as the first line.
+
+* Each line can only contain a single argument.
+
+* The same style applies to dicts, lists, return calls, etc.
```

Comment:
Most of the listings have an extra line in between.
And it helps with reformating - one line per sentence, if you have an editor command to do that.